vendor/plugins/edu.harvard.dce.paella.heartbeatSender/heartbeat_sender.js

```diff
@@ -36,6 +36,15 @@ Class(
           10
         );
 
+        // In the case of a live stream and a config setting that says to not 
+        // play on load, paella.player.videoContainer.paused() will always 
+        // return true, so it's not reliable then.
+        // However, our live stream player does not allow pausing. If you are 
+        // watching live stream, you are playing. So, we can count on that to 
+        // determine play state.
+        var isPlaying = !paella.player.videoContainer.paused() ||
+          paella.player.isLiveStream();
+
```

Comment:
That's the event that would set isPlaying to rube, but is blocked by the _playOnLoad == false.